dissect/target/tools/diff.py

```diff
@@ -0,0 +1,907 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import annotations
+
+import argparse
+import dataclasses
+import logging
+import re
+import shutil
+import sys
+from difflib import diff_bytes, unified_diff
+from fnmatch import fnmatch, translate
+from io import BytesIO
+from typing import Iterable, Iterator, TextIO
+
+from flow.record import (
+    IGNORE_FIELDS_FOR_COMPARISON,
+    Record,
+    RecordOutput,
+    set_ignored_fields_for_comparison,
+)
```

Comment:
We can now use the `ignore_fields_for_comparison` context manager.

```suggestion
from flow.record import (
    ignore_fields_for_comparison,
    Record,
    RecordOutput,
)
```